lib/puppet/rest/routes.rb

```diff
@@ -1,4 +1,5 @@
 require 'puppet/rest/route'
+require 'time'
```

Comment:
Why do we need this?

Reply:
`Time` is part of Ruby core, though the 'date' module is part of the stdlib and monkey patches `Time`, as well as provides new classes like `DateTime`. From the docs it seems that `httpdate` is defined on a DateTime instance not on a time instance.

from my console:
```
irb(main):001:0> t = Time.now
=> 2018-06-26 10:32:49 -0700
irb(main):002:0> t.httpdate
NoMethodError: undefined method `httpdate' for 2018-06-26 10:32:49 -0700:Time
        from (irb):2
        from /Users/justin/.rubies/ruby-2.3.4/bin/irb:11:in `<main>'
irb(main):003:0> require 'date'
=> true
irb(main):004:0> tt = Time.now
=> 2018-06-26 10:33:04 -0700
irb(main):005:0> tt.httpdate
NoMethodError: undefined method `httpdate' for 2018-06-26 10:33:04 -0700:Time
        from (irb):5
        from /Users/justin/.rubies/ruby-2.3.4/bin/irb:11:in `<main>'
irb(main):006:0> dt = DateTime.now
=> #<DateTime: 2018-06-26T10:33:16-07:00 ((2458296j,63196s,884349000n),-25200s,2299161j)>
irb(main):007:0> dt.httpdate
=> "Tue, 26 Jun 2018 17:33:16 GMT"
```

Reply:
for the .httpdate
Let me confirm

Reply:
I'd only ever seed folks require datetime for this kind of behvior, Amy showed me this is actually a better usage.